liquidctl/config/load.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+import sys
+
+def get_config_files(file=None, appname='liquidctl', **kwargs):
+    """
+    This will get all the potential file paths on the system for where the config file
+    can be
+
+    This will also handle the -f overrides
+    """
+    files = []
+    if file:
+        files.append(file)
+
+    if sys.platform == 'win32':
+        files.append(os.path.join(os.getenv('APPDATA'), appname, 'config.toml'))
+    elif sys.platform == 'darwin':
+        files.append(os.path.expanduser(os.path.join('~/Library/Application Support', appname, 'config.toml')))
+        files.append(os.path.expanduser(os.path.join('~', f'.{appname}.toml')))
+    elif sys.platform == 'linux':
+        XDG_CONFIG_HOME = os.getenv('XDG_CONFIG_HOME')
+        XDG_CONFIG_DIRS = os.getenv('XDG_CONFIG_DIRS')
+
+        # treat all other platforms as *nix and conform to XDG basedir spec
+        if XDG_CONFIG_HOME:
+            files.append(os.path.join(XDG_CONFIG_HOME, appname, 'config.toml'))
+
+        files.append(os.path.expanduser(os.path.join('~', '.config', appname, 'config.toml')))
+        files.append(os.path.expanduser(os.path.join('~', f'.{appname}.toml')))
+
+        if XDG_CONFIG_DIRS:
+            files.append(os.path.join(XDG_CONFIG_DIRS, appname, 'config.toml'))
```

Comment:
XDG_CONFIG_DIRS should come first, and if it's not present, it should default to `/etc`.

Reply:
did I mix up the orders of these? isnt `XDG_CONFIG_DIRS=/etc` and `XDG_CONFIG_HOME=~/.config`? if so that is the order that we want, the specific user folder is first and then it should fall back to the system specific ones. 

Ill re read the XDG spec again over the weekend, maybe I read it wrong.

Reply:
You're right, my bad.